llvm/test/Analysis/DependenceAnalysis/PreliminaryNoValidityCheckFixedSize.ll

```diff
@@ -20,7 +20,7 @@ define void @p2(i64 %n, ptr %A, ptr %B) nounwind uwtable ssp {
 ; CHECK-NEXT:  Src: store i64 %i.011, ptr %arrayidx8, align 8 --> Dst: store i64 %i.011, ptr %arrayidx8, align 8
 ; CHECK-NEXT:    da analyze - none!
 ; CHECK-NEXT:  Src: store i64 %i.011, ptr %arrayidx8, align 8 --> Dst: %0 = load i64, ptr %arrayidx17, align 8
-; CHECK-NEXT:    da analyze - flow [-3 -2]!
+; CHECK-NEXT:    da analyze - flow [-3 -2 / -1]!
```

Comment:
Rather than merging two DA interpretations into the same output, have you considered keeping the separate for better understandability. E.g.:
```
da analyze - flow [-3 -2]! / assuming 1 fused loop: [-3 -2 -1]!
```
Because of how FileCheck works, this test wouldn't even needed to be changes.